code/src/main/java/com/onlinecontacttracing/servlets/CustomMessageAndContactsServlet.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.servlets;
+
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Load;
+import com.googlecode.objectify.Objectify; 
+import com.googlecode.objectify.ObjectifyFactory; 
+import com.googlecode.objectify.ObjectifyService; 
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUserWithMessage;
+import com.onlinecontacttracing.messsaging.GeneratedUserId;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.lang.ClassCastException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Map;
+/**Servlet to put custom message data into Ofy store */
+@WebServlet("/customMessage")
+public class CustomMessageAndContactsServlet extends HttpServlet {
+  private final ArrayList<String> employeeComment = new ArrayList<String>();
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+  /**
+   * Adds the Emails and custom message to the Objectify store
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println(request.getParameterMap().toString());
+
+    int numberOfRecipients = 0;
+    try {
+      numberOfRecipients = Integer.parseInt(request.getParameter("number-of-recipients-box"));
+    } catch(NumberFormatException e) {
+        System.err.println("Not a number");
+        numberOfRecipients = -1;
+    }
+    ArrayList<String> emailAddresses = new ArrayList<String>();
+    for(int i  = 0; i < numberOfRecipients; i++) {
+      emailAddresses.add(request.getParameter("email-box-" + (i + 1)));
+    }
+
+    String userId = new GeneratedUserId("anon-").getIdString();
```

Comment:
No, I mostly did it this way in case someone some future programmer wanted to use the code without sticking anon to the front but if you think it'd be better then i can change it

Reply:
That's fine with me. If you can think of possible use-cases without "anon-" at the front than doing it this way is better, I agree.